<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>channel_widget</class>
 <widget class="QWidget" name="channel_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove titlebar</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="mode_combobox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <item>
     <property name="text">
      <string>looped</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>random</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>once</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="channel_mute">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-no-audio-30.png</normaloff>icons/icons8-no-audio-30.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="channel_play">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-play-30.png</normaloff>icons/icons8-play-30.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="balance_label">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>L           R</string>
    </property>
   </widget>
   <widget class="QDial" name="channel_balance">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>23</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="channel_label">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>0</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fireplace.mp3</string>
    </property>
   </widget>
   <widget class="QComboBox" name="time_unit_box">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1 min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 hour</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="amount_spinbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>88</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="suffix">
     <string> times</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSlider" name="channel_volume">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>25</y>
      <width>20</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>20</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
